config/services/buckets-mdapi/service.json.lab

```diff
@@ -1,5 +1,8 @@
 {
 	"params": {
 		"ram": 1024
+	},
+	"metadata": {
+		"BUCKETS_MDAPI_CUEBALL_MAX_CONNECTIONS": 64
```

Comment:
Khang *had* suggested 48 here in the description for MANTA-4978, and then you said "16" for lab in a comment:

> For buckets-mdapi, we can make the number of connections similar to those of moray, i.e. 16 for coal/lab and 64 (default) for production.

Reply:
The latest is now 32 for coal, 64 for lab, and 64 for production. Looking at the number of mdapi instances we'd normally provisioned per shard and the postgresql settings he referenced for different sizes:

postgresql.manta.coal.conf:max_connections = 100 (1 mdapi per shard)
postgresql.manta.lab.conf:max_connections = 210 (1-2 mdapi per shard)
postgresql.manta.production.conf:max_connections = 1000 (3-6 mdapi per shard)

we can actually keep the "lab" one to 64.